### PR TITLE
chore: adds nightly tests both for coraza and caddy

### PR DESCRIPTION
**Pull request by jcchavezs**

### CODE REVIEW

File: .github/workflows/tests.yml [hunk 1]
```diff
@@ -35,6 +36,6 @@ jobs:
       - name: Install xcaddy
         run: go install github.com/caddyserver/xcaddy/cmd/xcaddy@${{ matrix.xcaddy-version }}
       - name: Build caddy
-        run: go run mage.go buildLinux
+        run: CADDY_VERSION=${{ matrix.caddy-version }} go run mage.go buildLinux
```
Comment (jcchavezs) [inline .github/workflows/tests.yml:39]:
```suggestion
        run: CADDY_VERSION=${{ matrix.caddy-version }} go run mage.go buildCaddyLinux
```

**Comment by jcchavezs** [comment]
I wonder which versions are reasonable to support @mholt

**Comment by mholt** [comment]
We really only support the latest, esp. the in the minor tree; i.e. 2.6.4 currently.

**Comment by jcchavezs** [comment]
Thanks. That makes sense.

**Review by fzipi — APPROVED** [review]
LGTM.

**Pull request merged**